### DISCUSSION TITLE
How to escape curly brackets or braces?

### DISCUSSION THREAD

**Discussion by titocosta** [amannn/next-intl#259] [Q&A] [answered]
I need curly brackets `{` `}` inside my message. How do I escape it to avoid confusion and errors related to variable substitution?

**Comment by amannn** [comment, marked as answer]
Hey! ICU uses single quotes as the escape character.

```
"Escape curly braces with single quotes (e.g. '{name'})"
```

I'm also adding this to the docs, thank you for mentioning this!

**Reply by ZehuaZhang** [reply]
I have a use case wanting to have a `'` before `{}`, how do I achieve to escape single quote itself
The translated example: View 'Content' in 'Page 4'

**Reply by AndreasGiersch** [reply]
You can just use double single quotes, e.g. `View ''{textKey}'' in ''{page}'' ....`
